functions/menu.bash

```diff
@@ -131,9 +131,10 @@ show_main_menu() {
     "37 | Move root to USB"       "Move the system root from the SD card to a USB device (SSD or stick)" \
     "38 | Use ZRAM"               "Use compressed RAM/disk sync for active directories to avoid SD card corruption" \
     "   | Uninstall ZRAM"         "Don't use compressed memory (back to standard Raspberry Pi OS filesystem layout)" \
-    "39 | Setup VPN access"       "Setup Wireguard to enable secure remote access to openHABian (BETA)" \
-    "   | Remove Wireguard VPN"   "Remove Wireguard VPN from openHABian" \
-    "3A | Setup Exim Mail Relay"  "Install Exim4 to relay mails via public email provider" \
+    "39 | Setup Exim Mail Relay"  "Install Exim4 to relay mails via public email provider" \
+    "3A | Setup tailscale VPN"    "Establish or join a WireGuard based VPN using the tailscale service (BETA)" \
```

Comment:
Where is the option to remove tailscale from the menu? Does the wireguard remove remove tailscale as well?

Reply:
no, I added it